test/data/users.js

```diff
@@ -0,0 +1,52 @@
+module.exports = {
+  champion1: {
```

Comment:
Yeah, not a bad idea, since this is the same data as is used when creating the testdata.